tests/github/w3c/permissions/index.html

```diff
@@ -641,9 +641,8 @@ <h2 class="no-num no-toc no-ref heading settled" id="subtitle"><span class="cont
      <dd><a href="https://www.w3.org/TR/permissions/">https://www.w3.org/TR/permissions/</a>
      <dt>Previous Versions:
      <dd><a href="https://www.w3.org/TR/2015/WD-permissions-20150407/" rel="prev">https://www.w3.org/TR/2015/WD-permissions-20150407/</a>
-     <dd><a href rel="prev"></a>
      <dt>Feedback:
-     <dd><span><a href="mailto:public-webappsec@w3.org?subject=%5Bpermissions%5D%20YOUR%20TOPIC%20HERE">public-webappsec@w3.org</a> with subject line “<kbd>[permissions] <i data-lt>… message topic …</i></kbd>”</span>
+     <dd><span><a href="mailto:public-webappsec@w3.org?subject=%5Bpermissions%5D%20YOUR%20TOPIC%20HERE">public-webappsec@w3.org</a> with subject line “<kbd>[permissions] <i data-lt>… message topic …</i></kbd>” (<a href="http://lists.w3.org/Archives/Public/public-webappsec/" rel="discussion">archives</a>)</span>
```

Comment:
This may be the wrong format, since it's pulling in an extra metadata element `Mailing List Archives`, but that was missing before

Reply:
No, it's correct, the blank line was just hiding this metadata before.